scripts/classes/app-binary-configurator.js

```diff
@@ -144,30 +146,33 @@ class AppBinaryConfigurator {
   }
 
   configureAppInfoAndroid() {
-    console.log('Configuring build.gradle');
+    console.log('Configuring', 'build.gradle'.bold);
     const buildGradlePath = './android/app/build.gradle';
     const buildGradle = fs.readFileSync(buildGradlePath, 'utf8');
-    const applicationId = this.publishingProperties.android_market_package_name;
     const newBuildGradle = buildGradle
-      .replace(/\sapplicationId\s.*/g, ` applicationId '${applicationId}'`)
       .replace(/\sversionCode\s.*/g, ` versionCode ${this.getBinaryVersionCode()}`)
       .replace(/\sversionName\s.*/g, ` versionName '${this.getBinaryVersionName()}'`)
       .replace(/ShoutemApplicationName/g, this.publishingProperties.android_name);
     fs.writeFileSync(buildGradlePath, newBuildGradle);
   }
 
   configureAppInfo() {
-    if (this.config.platform === 'ios') {
-      this.configureAppInfoIOS();
-    } else if (this.config.platform === 'android') {
-      this.configureAppInfoAndroid();
-    }
+    this.configureAppInfoIOS();
+    this.configureAppInfoAndroid();
+  }
+
+  runForAllPlatforms(configureFunction) {
+    return Promise.all(_.map(binarySettings, (settings, platform) => {
+      if (_.isFunction(configureFunction)) {
+        configureFunction(settings, platform);
+      }
+    }));
   }
 
   configureApp() {
     return this.getPublishingProperties()
-      .then(() => this.configureLaunchScreen())
-      .then(() => this.configureAppIcon())
+      .then(() => this.runForAllPlatforms(this.configureLaunchScreen))
+      .then(() => this.runForAllPlatforms(this.configureAppIcon))
       .then(() => this.configureAppInfo());
```

Comment:
It is a bit weird that all other configurations are executed through `runForAllPlatforms` except for this one...